Casks/apple-store-time-machine.rb

```diff
@@ -1,6 +1,6 @@
 cask "apple-store-time-machine" do
```

Comment:
Following the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference) guidelines in the cask cookbook -

```suggestion
cask "shop-different" do
```

The filename would need to be also updated to `shop-different.rb`